AB_environments/AB_baseline.conda.yaml

```diff
@@ -44,7 +44,10 @@ dependencies:
   - gilknocker ==0.4.1
   - openssl >1.1.0g
   - pyspark ==3.4.1
-  - openjdk ==20.0.2
+  - openjdk ~=11.0
+  - grpcio ==1.58.1
+  - grpcio-status ==1.57.0
+  - protobuf ==4.24.3
```

Comment:
Please sort alphabetically :)

Reply:
FWIW this whole file is very unalphabetical.

Reply:
within this group? Because nothing else in this file is sorted alphabetically.